pxtlib/util.ts

```diff
@@ -1201,30 +1207,41 @@ namespace ts.pxtc.Util {
         "is": { englishName: "Icelandic", localizedName: "Íslenska" },
         "it": { englishName: "Italian", localizedName: "Italiano" },
         "ja": { englishName: "Japanese", localizedName: "日本語" },
+        "ja-hira": {englishName: "Hiragana", localizedName: "にほんご"},
```

Comment:
```suggestion
        "ja-HIRA": { englishName: "Hiragana", localizedName: "にほんご" },
```

quick update for consistency, we changed the language code in crowdin already~

Reply:
Might be a digression but to be precise with RFC 5646, it's one of:

- `language code` + `region`
- `language code` + `script`
- `language code` + `script` + `region`.

Hiragana in this case is a 'script' so it's commonly expressed as `ja-Hira`, whereas the region codes are all upper-case.

Refs:

- https://www.w3.org/International/articles/language-tags/
- https://www.iso.org/schema/isosts/v1.0/doc/n-cvd0.html

Reply:
So should I fix to `ja-Hira`?

Reply:
No, not at the moment. That will require a corresponding change the cloud serving stuff so we'll leave it as-is unless others on the team want to use this format.